test/acceptance/tests.py

```diff
@@ -63,13 +63,26 @@ class OpenAssessmentTest(WebAppTest):
         'file_upload':
             u'courses/{test_course_id}/courseware/'
             u'57a3f9d51d424f6cb922f0d69cba868d/bb563abc989340d8806920902f267ca3/'.format(test_course_id=TEST_COURSE_ID),
+        'full_workflow':
+            u'courses/{test_course_id}/courseware/'
+            u'676026889c884ac1827688750871c825/181ea9ff144c4766be44eb8cb360e34f/'.format(test_course_id=TEST_COURSE_ID),
     }
 
     SUBMISSION = u"This is a test submission."
     LATEX_SUBMISSION = u"[mathjaxinline]( \int_{0}^{1}xdx \)[/mathjaxinline]"
     OPTIONS_SELECTED = [1, 2]
     STAFF_OVERRIDE_OPTIONS_SELECTED = [0, 1]
+    STAFF_OVERRIDE_SCORE = 1
+    STAFF_OVERRIDE_EXISTS = "COMPLETE"
+    STAFF_OVERRIDE_LEARNER_STEPS_NOT_COMPLETE = "YOU MUST COMPLETE THE STEPS ABOVE TO VIEW YOUR GRADE"
+    STAFF_OVERRIDE_STAFF_AREA_SCORE = "Final grade: 1 out of 8"
```

Comment:
Can you make this a parameterized string so there can be other tests with a different final grade? I think it would also make the test clearer.